Magento.gitignore

```diff
@@ -45,8 +45,43 @@ app/etc/local.xml
 app/.htaccess
 app/locale/
 app/Mage.php
+app/bootstrap.php
 /cron.php
 cron.sh
+/dev/tests/functional/etc
+/dev/tests/functional/lib/Magento/Mtf
+/dev/tests/functional/tests/app/Mage
+/dev/tests/functional/testsuites/Magento/Mtf/TestSuite/InjectableTests/3rdParty.xml
```

Comment:
Like the name already suggested you can use these xml files to describe test suites. By default the basic.xml is used. But you could optionally change this behavior via the configuration file. And it seems the developers at Magento already give a away some of there own integration test suites.

So yes, this a place where you add your own testsuites. Which make it hard to catch it in a wildcard without catching too much.